qiskit/providers/aer/noise/errors/readout_error.py

```diff
@@ -127,12 +127,12 @@ def atol(self, atol):
         self.ATOL = atol
```

Comment:
For base operators these update the *class* attributes, not the object, so this should be
```suggestion
        ReadoutError.ATOL = atol
```